products/cloudrun/api.yaml

```diff
@@ -540,6 +540,10 @@ objects:
                 the default value.
             - `1` not-thread-safe. Single concurrency
             - `2-N` thread-safe, max concurrency of N
+        - !ruby/object:Api::Type::Integer
+          name: timeoutSeconds
+          description: |-
+            TimeoutSeconds holds the max duration the instance is allowed for responding to a request. Not currently used by Cloud Run.
```

Comment:
I think the sentence "Not currently used by Cloud Run." is more confusing than helpful in this context, and should probably be removed.

Reply:
fixed, but it's take from https://cloud.google.com/run/docs/reference/rest/v1/RevisionSpec?hl=ro

Reply:
Yeah, it's totally fine to modify descriptions for TF docs so that they make more sense. Using the ones from the GCP docs is just handy as a starting point.